packages/plans-grid-next/src/components/shared/header-price/index.tsx

```diff
@@ -14,12 +21,27 @@ interface HeaderPriceProps {
 	visibleGridPlans: GridPlan[];
 }
 
+const getTermVariantPlanSlugForSavings = ( {
+	planSlug,
+	billingPeriod,
+}: {
+	planSlug: PlanSlug;
+	billingPeriod?: -1 | ( typeof PERIOD_LIST )[ number ];
+} ) => {
+	if ( billingPeriod && 365 <= billingPeriod ) {
```

Comment:
`Non-blocking / nitpick:`

It may be worth leaving an inline comment explaining why we're using monthly term for savings. It took me a little bit to wrap my head around what was happening here until I dug into your comment [below](https://github.com/Automattic/wp-calypso/pull/96357/files#r1842493952).

Reply:
Good idea